compiler/rustc_parse/src/parser/diagnostics.rs

```diff
@@ -613,6 +613,17 @@ impl<'a> Parser<'a> {
             }
         }
 
+        if let TokenKind::Ident(symbol, _) = &self.prev_token.kind {
+            if ["def", "fun", "func", "function"].contains(&symbol.as_str()) {
+                err.span_suggestion_short(
+                    self.prev_token.span,
+                    &format!("write `fn` instead of `{symbol}` to declare a function"),
```

Comment:
Is this missing the `symbol` format?

Reply:
I am a bit confused. What do you mean?

Reply:
If you were expecting something like `&format!("stuff `{}` more stuff", symbol)`, you can embed identifiers inside `{}`.

Reply:
Oh I'm an idiot, my bad